mlrun/feature_store/feature_set.py

```diff
@@ -812,6 +827,14 @@ def to_dataframe(
             if self.spec.timestamp_key and self.spec.timestamp_key not in entities:
                 columns = [self.spec.timestamp_key] + columns
             columns = entities + columns
+
+        if self.spec.passthrough:
+            if not self.spec.source:
+                raise mlrun.errors.MLRunNotFoundError(
+                    "there are no offline targets for this feature set"
```

Comment:
Might be a bit confusing without mentioning passthrough.

Reply:
fixed